lib/samly/trace.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Samly.Trace do
+  def handle(message, data \\ %{}) do
+    case Application.get_env(:samly, Samly.Trace, []) do
```

Comment:
That third parameter should be `nil`, surely? (Which is the default, so it doesn't need to be there at all).

Reply:
Oh, yes